scripts/dev/find-fix.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import argparse, os, re, subprocess
+
+script_path = os.path.dirname(os.path.realpath(__file__))
+root_dir = os.path.dirname(os.path.dirname(os.path.dirname(script_path)))
+
```

Comment:
Do you need to include acs-community-packaging too?

Reply:
I didn't include acs-community-packaging, but there's no real reason not to.  I think it should be easy to update the script to include this too.